pkg/kotscli/kotscli.go

```diff
@@ -25,7 +26,8 @@ type InstallOptions struct {
 }
 
 func Install(opts InstallOptions, msg *spinner.MessageWriter) error {
```

Comment:
just a reminder that KOTS puts stuff in the tmp directory, and that needs to be consolidated as well

Reply:
This pr does not address the /tmp dir. i will do that in a follow up

Reply:
Done